integration_tests/rest_api/setup_data_hooks.py

```diff
@@ -183,6 +183,14 @@ def add_holding(txn):
             'targetQuantity': 1337
         })
 
+@hooks.before('/offers/{id}/accept > PATCH > 200 > application/json')
+def add_accept_info(txn):
+    patch_body(txn, {
+            'source': seeded_data['offer']['source'],
+            'sourceQuantity': seeded_data['offer']['sourceQuantity'],
+            'target': seeded_data['offer']['target']
```

Comment:
The only reason this works is because the offer was set up with what happens to be same asset type for both the source and the target. Remember that these terms are relative to the resource they are attached to. An offer's source is where it will pay out from. An AcceptOffer's source is where _it_ will pay out from.

```
OFFER       ACCEPT OFFER
source  ->  target
target  <-  source
```

Part of the point of tests is to be a reference, so we should probably have two types of assets in four holdings under two accounts with everything lined up properly.

However, for now I would settle for now for just switching target and source around.